api/tests.py

```diff
@@ -1,3 +1,53 @@
-from django.test import TestCase
+from datetime import datetime, timedelta
+import json
 
-# Create your tests here.
+from django.test import RequestFactory, TestCase
+
+from .views import reminders
+from alerts.models import Alert
+
+
+class testReminders(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def _post(self, url, params):
+        return self.factory.post(url, params)
+
+    def testReminderWithArraignmentIn8Days(self):
+        arraignment_datetime = (datetime.today() + timedelta(days=8)).strftime('%Y-%m-%dT%H:%M:%S')
+        request = self._post('/api/reminders', {
+            "arraignment_datetime": arraignment_datetime,
+            "case_num": 1,
+            "phone_num": "+1-918-555-5555",
+        })
+        response = reminders(request)
+        resp_json = json.loads(response.content)
+        self.assertEqual(Alert.objects.all().count(), 2)
+        self.assertEqual(resp_json['status'], '201 Created')
+
+
+    def testReminderWithArraignmentIn2Days(self):
+        arraignment_datetime = (datetime.today() + timedelta(days=2)).strftime('%Y-%m-%dT%H:%M:%S')
+        request = self._post('/api/reminders', {
+            "arraignment_datetime": arraignment_datetime,
+            "case_num": 2,
+            "phone_num": "+1-918-555-5555",
+        })
+        response = reminders(request)
+        resp_json = json.loads(response.content)
+        self.assertEqual(Alert.objects.all().count(), 1)
+        self.assertEqual(resp_json['status'], '201 Created')
+
+
+    def testReminderWithArraignmentToday(self):
+        arraignment_datetime = datetime.today().strftime('%Y-%m-%dT%H:%M:%S')
+        request = self._post('/api/reminders', {
+            "arraignment_datetime": arraignment_datetime,
+            "case_num": 3,
+            "phone_num": "+1-918-555-5555",
+        })
+        response = reminders(request)
+        resp_json = json.loads(response.content)
+        self.assertEqual(Alert.objects.all().count(), 0)
+        self.assertEqual(resp_json['status'], '410 Gone')
```

Comment:
Could add another test ...

```
def testDuplicateReminder(self):
```

Which creates an `Alert` and then tries to use the `reminders` view to create another `Alert` with the same datetime, case_num, and phone_num.

To catch it, could use `Alert.objects.get_or_create()` in the `api/views.py`.